### PR TITLE
Add Hungarian locale

### PR DESCRIPTION
**Pull request by urbalazs**

### CODE REVIEW

**Comment by ghostwords** [comment]
Hello and thank you! I should have some feedback tomorrow. In the meantime, you could join [Privacy Badger's Hungarian language team on Transifex](https://www.transifex.com/eff/privacy-badger/language/hu/).

File: src/_locales/hu/messages.json
Comment (ghostwords) [inline src/_locales/hu/messages.json:397]: Does this say "drag" or "move"? It should say "move", as one can't actually drag Privacy Badger's sliders, only move/reposition them by clicking (or tapping).

File: src/_locales/hu/messages.json
Comment (ghostwords) [inline src/_locales/hu/messages.json:459]: Does this say "drag" or "move"? It should say "move", as one can't actually drag Privacy Badger's sliders, only move/reposition them by clicking (or tapping).

File: src/_locales/hu/messages.json
Comment (ghostwords) [inline src/_locales/hu/messages.json:405]: "suggestions and similar pages" or "suggestions for similar pages"? Should be "for". The suggestions *are* the similar pages, not a distinct thing.

**Review by ghostwords — COMMENTED** [review]
Thank you again! The translations look great generally. Just a few nits and questions.

**Comment by urbalazs** [comment]
@ghostwords thank you for the feedback, I updated the Hungarian locale based on your suggestions.

I left "nem haszonszerzésért" for the translation of "nonprofit" in the mentioned sentence. The "nonprofit foundation" ("nonprofit alapítvány" in Hungarian) is a well known term, but the "nonprofit fight" is not known in Hungary. That's why I translated the word "nonprofit" in this sentence.

Just for curiosity: do you speak Hungarian, did you ask somebody who is speaking Hungarian to review my translation, or did you use machine translation for the review?

**Comment by ghostwords** [comment]
Thank you! I'll take a proper look on Monday.

>nonprofit fight

To be clear, "nonprofit" there is a noun, not an adjective. As in: "We are a **[nonprofit organization/foundation]** fighting for your rights ...". We should be consistent with how we translate nonprofit in both messages that use "nonprofit".

>Just for curiosity: do you speak Hungarian, did you ask somebody who is speaking Hungarian to review my translation, or did you use machine translation for the review?

Machine translation (Google Translate). I've been reviewing Privacy Badger translations for a few years now, so I have an idea as to the kinds of translation problems we tend to run into.

Are you interested in joining the Hungarian team for Privacy Badger on Transifex?

**Comment by urbalazs** [comment]
> To be clear, "nonprofit" there is a noun (short for "nonprofit organization"), not an adjective. As in: "We are a **[nonprofit organization/foundation]** fighting for your rights ...". We should be consistent in how we translate "nonprofit" in both messages that use "nonprofit".

OK, I understand now. "Nonprofit" has now consistent translation.

> Are you interested in joining the Hungarian team for Privacy Badger on Transifex?

Yes, sure! I have already requested to join, and I'm waiting for approval. It would be great to promote me as coordinator (or at least reviewer). Thanks!

**Review by ghostwords — APPROVED** [review]
Thanks again!

**Comment by ghostwords** [comment]
I have a minor thing or two, but let's continue on Transifex.

**Pull request merged**
